releasenotes/notes/27984.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+  - 27984
+
+# releaseNotes is a markdown listing of any user facing changes. This will appear in the
+# release notes.
+releaseNotes:
+- |
+  **Added** Enable regex matching for authorization policy
```

Comment:
Make it clear that the feature is disabled by default, and can be enabled by setting the global flag on istiod.

Reply:
modified. Thanks. Let me know if you would like to change the content.